### PR TITLE
Add initial unit tests for handler and result, made changes accordingly

### PR DESCRIPTION
**Pull request by ljades**
**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR:
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name: @okeefm
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name: @holmesie
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

**Review by okeefm — CHANGES_REQUESTED** [review]
Need to add `mongoose` to `package.json`

**Review by okeefm — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -18,5 +18,8 @@
   "homepage": "https://github.com/projectcypress/js-ecqm-engine#readme",
   "devDependencies": {
     "jasmine": "^2.9.0"
+  },
+  "dependencies": {
+    "mongoose": "^5.0.3"
```
Comment (okeefm) [inline package.json:20]: Can you commit the resultant `package-lock.json` from running `npm install` here? It should be commited per https://docs.npmjs.com/files/package-lock.json

**Review by okeefm — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (holmesie) [reply package.json:20]: @okeefm Just a thought, have we considered using Yarn for js dependency management? Not a biggie, but I'm not sure we ever talked about dependency management.

File: package.json [hunk 2]
```diff
@@ -6,17 +6,17 @@
...
     "test": "jasmine"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/projectcypress/js-ecqm-engine.git"
-  },
-  "author": "",
-  "license": "ISC",
+  "repository": "git+https://github.com/projectcypress/js-ecqm-engine.git",
+  "author": "Louis Ades <lades@mitre.org>",
```
Comment (okeefm) [inline package.json]: We should probably list all of us here, something like
    ```"contributors": ["Louis Ades <lades@mitre.org>",
                   "Michael O'Keefe <mokeefe@mitre.org>",
                   "Adam Holmes <aholmes@mitre.org>"],```

File: package.json [hunk 3]
```diff
@@ -6,17 +6,17 @@
...
+  "license": "Apache 2",
```
Comment (okeefm) [inline package.json]: change `"Apache 2"` to `"Apache-2.0"` to be SPDX-compliant.

**Review by okeefm — CHANGES_REQUESTED** [review]

**Review by okeefm — APPROVED** [review]
Looks good to me. Thanks for putting up with the feedback!

**Review by holmesie — APPROVED** [review]
Looks good!
